### PR TITLE
Improving tests

### PR DESCRIPTION
**Pull request by hosein2398**
I'm also wondering how we can write some test for CSS files, if we do it then it's like nearly most features and conditions are tested.
Look at the latest commit please, see if you can fix it.

### CODE REVIEW

**Comment by hosein2398** [comment]
ping @egoist :)

File: src/index.js [hunk 1]
```diff
@@ -44,11 +44,7 @@ process.env.BUILD = 'production'
...
   static generate(options) {
-    try {
-      return new Bili(options).bundle({ write: false })
-    } catch (err) {
-      handleError(err)
-    }
+    return new Bili(options).bundle({ write: false })
   }
 
   static async write(options) {
```
Comment (egoist) [inline src/index.js]: maybe remove `try ... catch` for `Bili.write` too, we can handle that in `cli.js` instead.

Comment (hosein2398) [reply src/index.js]: Sounds good, Won't it break anything?

Comment (hosein2398) [reply src/index.js]: OK I go for it or you go?

**Pull request merged**